developing_SoCCo.py

```diff
@@ -0,0 +1,471 @@
+
```

Comment:
@jonathanrocher: Ok, I just created a folder sandbox.  Moving the developing_SCCo.py there.